clientcontroller/babylon.go

```diff
@@ -274,7 +277,13 @@ func (bc *BabylonController) QueryFinalityProviderVotingPower(fpPk *btcec.Public
 		blockHeight,
 	)
 	if err != nil {
-		return 0, fmt.Errorf("failed to query BTC delegations: %w", err)
+		allowedErr := fmt.Sprintf("rpc error: code = Unknown desc = %s: unknown request", btcstakingtypes.ErrVotingPowerTableNotUpdated.Wrapf("height: %d", blockHeight).Error())
+		if strings.EqualFold(err.Error(), allowedErr) {
+			// if nothing was updated in the voting power table, it should consider as zero VP to start to send pub random
+			return 0, nil
+		}
+
+		return 0, fmt.Errorf("failed to query Finality Voting Power at Height: %w", err)
```

Comment:
I started to think we should not ignore this error because the error essentially means that there's no voting power for any fps yet. If we keep retrying, then eventually the error will be gone. If we ignore this error, we won't revisit the voting power for this height, which could lead to missing a block that the FP has voting power.

I think we should ensure that `QueryFinalityProviderVotingPower` would not be called before [waitForActivation](https://github.com/babylonlabs-io/finality-provider/blob/f7c17b17e6ec546ac9370b9a3d21686eeb5e12a7/finality-provider/service/chain_poller.go#L205) is finished.

Reply:
humm, this makes sense 
But I had understood that the finality provider should send pub rand list even if he does not have voting power, only the finality signature should verify that it has voting power, before sending 

Is my understanding correct?

Reply:
Yes. Thing is that if the fp gets `0` from `QueryFinalityProviderVotingPower`, it will skip sending finality votes for this height for good. In this PR, `QueryFinalityProviderVotingPower` can return `0` due to the protocol is not activated. So it is possible that `QueryFinalityProviderVotingPower` will return non-zero value for this height after the protocol is activated. I was worried that the fp might miss such heights due to this change

Reply:
humm, I think I understand 
For finality signatures we still want to respect this error and avoid skipping it! 
For send the pub rand list, we should still send it even if the btcstaking protocol is not activated yet

Reply:
modified at https://github.com/babylonlabs-io/finality-provider/pull/52/commits/a43b3f1865f7363630f9b69aa4112f51e4623056, now only the portion that updates from Created to Register, allows the error `ErrVotingPowerTableNotUpdated`